projects/storefrontstyles/scss/cxbase/mixins/_breakpoint.scss

```diff
@@ -0,0 +1,10 @@
+//breakpoint mixins - TODO maybe in another file?
```

Comment:
I think the mixins are really cool. Is there a reason for splitting them in two files?

Reply:
one is mixins and the others are functions - i saw separate files for them.
My question is: should we try to get rid of bootstrap breakpoints mixins/functions and use ours?

Reply:
Makes sense, perhaps we could use ours. The breakpoints are defined in the theme right? It could make it easier for clients to override them.